apps/condo/domains/banking/utils/validate/tin.utils.spec.js

```diff
@@ -0,0 +1,74 @@
+const { SPACE_SYMBOLS, SPACE_SYMBOL_LABLES } = require('@condo/domains/common/utils/string.utils')
+const { validateTin } = require('@condo/domains/banking/utils/validate/tin.utils')
+const { createValidRuTin10, createValidRuTin12 } = require('@condo/domains/banking/utils/testSchema/bankAccountGenerate')
+
+const SPACES = SPACE_SYMBOLS.split('')
+
+const COUNTRY_CODE_RU = 'ru'
+const VALID_RU_TINS = ['1654019570', '6311095616', '500110474504']
+const WRONG_LENGTH_TIN = '01234556789'
+const WRONG_FORMAT_TIN = '01234b567890'
+const INVALID_CONTROL_SUM_TIN_10 = '1234567810'
+const INVALID_CONTROL_SUM_TIN_12 = '500110474556'
+
+describe('validateTin', () => {
+    VALID_RU_TINS.forEach(tin => {
+        test(`for valid 10 or 12 char RU TIN (${tin})`, () => {
+            const { result } = validateTin(tin, COUNTRY_CODE_RU)
+            expect(result).toBe(true)
+        })
+
+        SPACES.forEach(spaceSymbol => {
+            test(`for valid 10 or 12 char RU TIN (${tin}) with spaces symbol (${SPACE_SYMBOL_LABLES[spaceSymbol] || spaceSymbol})`, () => {
+                const tinValue = `${spaceSymbol}${tin}${spaceSymbol}`
+
+                const { result } = validateTin(tinValue, COUNTRY_CODE_RU)
+                expect(result).toBe(true)
+            })
+        })
+    })
+
+    test('wrong length number as RU TIN', () => {
```

Comment:
You can group all these test cases into `describe('RU')` and avoid repeating `as RU` over and over.

Reply:
Done